robot.py

```diff
@@ -3,18 +3,24 @@
 import wpilib
 from ctre.cantalon import CANTalon
 from magicbot import MagicRobot
+import ctre
 
 from components.drivetrain import Drivetrain
+from components.intake import Intake
 
 
 class Robot(MagicRobot):
 
+    intake = Intake
     drivetrain = Drivetrain
 
     def createObjects(self):
         self.robot_drive = wpilib.RobotDrive(
             0, 1, 2, 3, motorController=CANTalon)
 
+        self.l_intake_motor = CANTalon(4)
```

Comment:
why is this still `l` and `r`?

Reply:
I changed it to master and slave when I had it in master-slave configuration but since reverse slave didn't work I changed it back to l and r. L and R make more sense now IMO than slave and master since they're no long slave and master.